thrift/src/main/thrift/atoms/media.thrift

```diff
@@ -17,7 +17,8 @@ enum Platform {
 
 enum AssetType {
   AUDIO = 0,
-  VIDEO = 1
+  VIDEO = 1,
+  SUBTITLES = 2
```

Comment:
To play devils argument https://www.w3schools.com/tags/att_track_kind.asp shows other 'kinds' of text we could have: 
```

captions | The track defines translation of dialogue and sound effects (suitable for deaf users)
chapters | The track defines chapter titles (suitable for navigating the media resource)
descriptions | The track defines a textual description of the video content (suitable for blind users)
metadata | The track defines content used by scripts. Not visible for the user
subtitles | The track defines subtitles, used to display subtitles in a video

```